install.sh

```diff
@@ -0,0 +1,405 @@
+#!/bin/bash
+
+# Call with:
+#   bash -c "$(curl -L https://rstd.io/r-install)"
+
+SCRIPT_ACTION=$1
+SCRIPT_ACTION=${SCRIPT_ACTION:-install}
+
+# Set to the full version to install. Must be either available on S3 or in the working directory
+R_VERSION=${R_VERSION:-}
+
+SUDO=
+if [[ $(id -u) != "0" ]]; then
+  SUDO=sudo
+fi
+
+# The root of the S3 URL for downloads
+CDN_URL='https://cdn.rstudio.com/r'
+
+# The URL for listing available R versions
+VERSIONS_URL="${CDN_URL}/versions.json"
+
+R_VERSIONS=$(curl ${VERSIONS_URL} | \
+  # Matches the JSON line that contains the r versions
+  grep r_versions | \
+  # Gets the value of the `r_version` property (e.g., "[ 3.0.0, 3.0.3, ... ]")
+  cut -f2 -d ":" | \
+  # Removes the opening and closing brackets of the array
+  cut -f2 -d "[" | cut -f1 -d "]" | \
+  # Removes the quotes and commas from the values
+  sed -e 's/\"//g' | sed -e 's/\,//g' | \
+  # Reverses the order of the list
+  ( while read -d ' ' f;do g="$f${g+ }$g" ;done;echo "$g" ))
```

Comment:
R 3.6.1 isn't showing up in the list of available versions, and I think it has to do with this line. For some reason, the read command is excluding the last item in the array?